src/c++/library/grpc_client.h

```diff
@@ -110,12 +110,15 @@ class InferenceServerGrpcClient : public InferenceServerClient {
   /// SSL encryption and authorization.
   /// \param keepalive_options Specifies the GRPC KeepAlive options described
   /// in https://grpc.github.io/grpc/cpp/md_doc_keepalive.html
+  /// \param force_new_connection If true, then a new channel is created for the
```

Comment:
If true, a new channel is created for each new client instance. When false, re-use old channels from cache for new client instances. The default value is false.